sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/TimeSeriesInsightsClient.cs

```diff
@@ -145,20 +145,19 @@ public virtual Response<TimeSeriesModelSettings> GetModelSettings(CancellationTo
         }
 
         /// <summary>
-        /// Updates model name or default type Id on Time Series model settings asynchronously.
+        /// Updates model name on Time Series model settings asynchronously.
         /// </summary>
-        /// <param name="options">Model settings update request body.</param>
+        /// <param name="name">Model display name which is shown in the UX and mutable by the user. Initial value is &quot;DefaultModel&quot;.</param>
```

Comment:
I wouldn't mention UX here as this API might not even be used by UX.

Reply:
Good observation, comes from swagger. Do you have any suggestions?